app-text/wordgrinder/wordgrinder-0.7.1.recipe

```diff
@@ -65,4 +95,11 @@ INSTALL()
 		LUA_PACKAGE=lua
 	mv $docDir/wordgrinder/README.wg $docDir
 	rmdir $docDir/wordgrinder
+	install -d $dataDir/wordgrinder/dictionaries
+	install -t $dataDir/wordgrinder/dictionaries extras/*.dictionary
+if [ -z "$secondaryArchSuffix" ]; then
+	packageEntries dictionaries \
+		$dataDir
+fi
+
```

Comment:
Just a cosmetic "change" :)
There's a blank line near the end of INSTALL (see the small red "-"). Could you move it 6 line up? (at the small green "+")
```diff
 	rmdir $docDir/wordgrinder
+
 	install -d $dataDir/wordgrinder/dictionaries
 	install -t $dataDir/wordgrinder/dictionaries extras/*.dictionary
 if [ -z "$secondaryArchSuffix" ]; then
 	packageEntries dictionaries \
 		$dataDir
 fi
-
 }
```

Reply:
OK, I understand why there was a misunderstanding. The comment above was not very clear, sorry.
I should have written:
* green plus sign = insert a blank line here.
* red minus sign = delete this blank line.